### PR TITLE
Fuzzy matching

### PR DESCRIPTION
**Pull request by ksingarajah**
Fuzzy matching mostly works, the threshold isn't fine-tuned, but right now it works for different casing and small deviations. For example:

screen shot 2018-08-03 at 4 35 53 pm

screen shot 2018-08-03 at 4 40 00 pm

### CODE REVIEW

File: src/main/java/codeu/bot/ActionMatcher.java [hunk 1]
```diff
@@ -78,6 +78,7 @@
 
     public ActionMatcher() {
         distance = new LevenshteinDistance();
+        System.out.println("init2");
```
Comment (tsalako) [inline src/main/java/codeu/bot/ActionMatcher.java]: do you want these logs in here?

**Review by tsalako — CHANGES_REQUESTED** [review]
I think there are other sections of the app that say "Chat App" or "Bottler" instead of "Botler" can you update these strings as well?

File: src/main/java/codeu/bot/ActionMatcher.java (hunk 1 above)
Comment (ksingarajah) [reply src/main/java/codeu/bot/ActionMatcher.java]: Deleted!

File: src/main/webapp/index.jsp [hunk 2]
```diff
@@ -26,10 +26,11 @@
       <div id="container">
         <div
           style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
-          <h1>CodeU Bottler</h1>
+          <h1>CodeU </h1>
```
Comment (tsalako) [inline src/main/webapp/index.jsp]: Do we want this to just say CodeU?

File: src/main/java/codeu/bot/ActionMatcher.java [hunk 3]
```diff
@@ -112,12 +113,12 @@ public String findFuzzyMatch(HashSet<String> set, int distanceThreshold) {
         for (String element : set) {
             for (Integer spaceIndex : spaceIndices) {
                 if (spaceIndex.intValue() + element.length() >= input.length()) {
+                    System.out.println("if statement");
```
Comment (tsalako) [inline src/main/java/codeu/bot/ActionMatcher.java:116]: do we want this log and the one below?

**Pull request merged**